brainkeeper/src/app/components/session_screen/small-person-card/small-person-card.component.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { Location } from '@angular/common';
+import { Router } from '@angular/router';
+import { SmallPersonCardComponent } from './small-person-card.component';
+import { Person } from 'src/app/classes/person';
+import { MatButtonModule } from '@angular/material/button';
+import { MatIconModule } from '@angular/material/icon';
+import { MatToolbarModule } from '@angular/material/toolbar';
+import { DebugElement } from '@angular/core';
+import { By } from '@angular/platform-browser';
+
+describe('SmallPersonCardComponent', () => {
+  let component: SmallPersonCardComponent;
+  let fixture: ComponentFixture<SmallPersonCardComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ SmallPersonCardComponent ],
+      imports: [
+        MatButtonModule,
+        MatToolbarModule,
+        MatIconModule
+      ],
+      providers: [
+        { provide: Location },
+        { provide: Router }
+      ],
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(SmallPersonCardComponent);
+    component = fixture.componentInstance;
+    component.person = new Person('Tatzelwurm', 'TatzelwurmSeinBild');
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+  it('should be no component when person is null', () => {
+    fixture.whenStable();
+    component.person = null;
+    fixture.detectChanges();
```

Comment:
I think here should also be a positive testcase testing whether the persons name and image is displayed

Reply:
I suppose you meant just the persons image, since the game would be fairly boring if the name of each possibility is displayed :). Changed with commit: 3c092315bd02c68198ed286b95baee75aafa7339